tests/test_package_manager.py

```diff
@@ -27,3 +30,30 @@ def test_set_index_urls():
     finally:
         manager.set_index_urls(default_index_urls)
         assert manager.index_urls == default_index_urls
+
+
+def test_freeze():
+    manager = test_package_manager()
```

Comment:
```suggestion
    manager = PackageManager()
```

calling a test function inside another test function seems a bit weird.

Reply:
True, I'll change `test_package_manager` to a non-test function.